### DISCUSSION TITLE
is it possible to do like this ?

### DISCUSSION THREAD

**Discussion by jaimeenkoffeekodes** [vercel/next.js#64483] [Help]
### Summary

![Screenshot 2024-04-15 105646](https://github.com/vercel/next.js/assets/149799052/39defba7-f7eb-4416-9f3e-f6d29d0589bd)

I want to add one more page in dynamic route.

**Comment by versecafe** [comment]
@jaimeenkoffeekodes what do you mean by extra page, you can do the following

`app/blog/[slug]/author`
and
`app/blog/[slug]/comments/[id]`

in the second case you can access both dynamic segments like this

```ts
interface PostProps {
  params: {
    slug: string;
    id: string;
  };
}
```

**Reply by jaimeenkoffeekodes** [reply]
Dhanyavad(Thanks)

**Comment by icyJoseph** [comment]
I guess you want `update_delivery_date/page.tsx`? Then yes.

And, if you want `update_delivery_data/[foo]/page.tsx`, then you'd just have more params in your server page entry component, or via useParams in client components.

**Reply by jaimeenkoffeekodes** [reply]
let me explain you the scenario(forgot my english): there is one deliveryPage where all the deliveries data are presented in table format, and in each row there is one option for edit and delete, so when I click on edit button, it should navigate me to [delivery_note_details] with id as slug, and from this details page, it should navigate to [update_page].
